sourcegraph/sourcegraph
# [SG-35151] [Accessibility]: Code monitoring logs should use list elements

### Audit type
ARC Toolkit

### User journey audit issue
[#34194](https://github.com/sourcegraph/sourcegraph/issues/34194)

### Problem description
- Navigate to https://sourcegraph.com/code-monitoring.
- Create some code monitors. Wait for some runs to happen so that you can view logs.
- Navigate to the "Logs" tab.

Code monitoring logs are displayed like a nested collapsible list, but are not implemented using list elements.

### Expected behaviour
- Top-level list of code monitors should be an `<ol>`. Timestamp of last run at that level should be right-aligned.
- List of code monitor runs ("Run X days/minutes ago") should be an `<ul>`.
- List of triggers and actions should be an `<ol>`.

### Additional Details
![167484261-a5773b74-2320-4480](https://user-images.githubusercontent.com/1741180/167484261-a5773b74-2320-4480-afaf-f363c2030bae.png)

## Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/35151)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-35151)

## Test plan

- Run `sg start web-standalone` on the terminal, ensure the localhost is running
- Go to http://localhost:3080/code-monitoring
- Go to the `Logs` tab
- Right click, select `Inspect Element` to inspect changes

### Loom video


## App preview:

- [Web](https://sg-web-contractors-sg-35151.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-wvhlbiqfhy.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
